cmake/onnxruntime_objectivec.cmake

```diff
@@ -42,15 +42,8 @@ set(OBJC_ROOT "${REPO_ROOT}/objectivec")
 # onnxruntime_objc target
 
 # these headers are the public interface
-# explicitly list them here so it is easy to see what is included
-set(onnxruntime_objc_headers
-    "${OBJC_ROOT}/include/onnxruntime.h"
-    "${OBJC_ROOT}/include/ort_coreml_execution_provider.h"
-    "${OBJC_ROOT}/include/ort_enums.h"
-    "${OBJC_ROOT}/include/ort_env.h"
-    "${OBJC_ROOT}/include/ort_session.h"
-    "${OBJC_ROOT}/include/ort_value.h"
-    )
+file(GLOB onnxruntime_objc_headers CONFIGURE_DEPENDS
+    "${OBJC_ROOT}/include/*.h")
```

Comment:
now that we have to specify this in two places, it's less maintenance to just match everything in the include directory